### PR TITLE
Feat/sharelocation

### PR DESCRIPTION
**Pull request by calilrenner**
As a logged in user, I want to share my location when making a post.

### CODE REVIEW

**Review by thipereira02 — COMMENTED** [review]
Screenshot from 2021-09-25 18-29-12
Posts are above the box with user's location

**Comment by calilrenner** [comment]
Already fixed zindex issue.

**Review by thipereira02 — APPROVED** [review]

File: src/pages/MyPosts.js [hunk 1]
```diff
@@ -24,6 +24,8 @@ export default function MyPosts() {
     });
   }, [onChangePost]);
 
+  console.log(userPosts);
```
Comment (sidneyouteiro) [inline src/pages/MyPosts.js:30]: Console.log perdido

File: src/components/CreateNewPost.js [hunk 2]
```diff
@@ -3,21 +3,37 @@ import styled from "styled-components";
...
 
     setButtonText("Publishing...");
     setDisabled(true);
 
-    const body = { link, text };
+    if(startLocation) {
+      body = { link, text, geolocation };
+    } else {
+      body = { link, text };
+    }
```
Comment (sidneyouteiro) [inline src/components/CreateNewPost.js:36]: `body = startLocation ? { link, text, geolocation } : { link, text };`

**Review by sidneyouteiro — APPROVED** [review]

**Pull request merged**